tests/integrations/python/tests/test_openai.py

```
@@ -3497,3 +3497,115 @@ def test_57_web_search_multi_turn_openai(self, provider, model, vk_enabled):
         print(f"✓ Second turn completed with {len(response2.output)} output items")
         print(f"✓ Multi-turn conversation test passed!")
 
+    # =========================================================================
+    # Async Inference Tests
+    # =========================================================================
+
+    @pytest.mark.parametrize("provider,model,vk_enabled", get_cross_provider_params_with_vk_for_scenario("simple_chat"))
+    def test_58_async_chat_completions(self, test_config, provider, model, vk_enabled):
```

Ruff (0.15.1)

[warning] Unused method argument: `test_config`

(ARG002)